WalletWasabi/Helpers/WasabiSignerHelpers.cs

```diff
@@ -0,0 +1,66 @@
+using NBitcoin.Crypto;
+using NBitcoin;
+using System.Threading.Tasks;
+using System.IO;
+using System.Security.Cryptography;
+
+namespace WalletWasabi.Helpers;
+
+public class WasabiSignerHelpers
+{
+	public static async Task SignSha256SumsFileAsync(string sha256SumsAscFilePath, Key wasabiPrivateKey)
+	{
+		var bytes = await File.ReadAllBytesAsync(sha256SumsAscFilePath).ConfigureAwait(false);
+		using SHA256 sha = SHA256.Create();
+		byte[] computedHash = sha.ComputeHash(bytes);
+
+		ECDSASignature signature = wasabiPrivateKey.Sign(new uint256(computedHash));
+
+		string base64Signature = Convert.ToBase64String(signature.ToDER());
+		var wasabiSignatureFilePath = Path.ChangeExtension(sha256SumsAscFilePath, "wasabisig");
+
+		await File.WriteAllTextAsync(wasabiSignatureFilePath, base64Signature).ConfigureAwait(false);
+	}
+
+	public static async Task VerifySha256SumsFileAsync(string sha256SumsAscFilePath)
+	{
```

Comment:
Maybe use `TryVerify..` practice, and return bool, and handle exception outside?

Reply:
AFAIK we want to fail hard if the signature is invalid.

Reply:
Makes sense.